pkg/kubelet/rkt/rkt.go

```diff
@@ -1424,6 +1504,20 @@ func (r *Runtime) SyncPod(pod *api.Pod, podStatus api.PodStatus, internalPodStat
 	return
 }
 
+func netnsPathFromUuid(uuid string) string {
+	return fmt.Sprintf("/var/run/netns/%s", uuid)
```

Comment:
It might be nice to prepend `k8-rkt-` or something so a user can easily identify namespaces we created. Fine with it as is as well.